src/content/docs/en/guides/images.mdx

```diff
@@ -475,6 +475,26 @@ export default defineConfig({
 });
 ```
 
+## Asset Caching
+
+Astro stores processed image assets in a cache directory during site builds, for both local and [remote images from authorized sources](/en/guides/images/#authorizing-remote-images). By preserving the cache directory between builds, processed assets are reused, improving build time and bandwidth usage.
+The default cache directory is `./node_modules/.astro`, however this can be changed using the `cacheDir` configuration value.
```

Comment:
```suggestion
Astro stores processed image assets in a cache directory during site builds, for both local and [remote images from authorized sources](/en/guides/images/#authorizing-remote-images). By preserving the cache directory between builds, processed assets are reused, improving build time and bandwidth usage.

The default cache directory is `./node_modules/.astro`, however this can be changed using the `cacheDir` configuration value.
```